src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -36,6 +36,34 @@ export default class SimidMapCreative extends BaseSimidCreative {
      * @private {?string}
      */
     this.searchQuery_ = null;
+    /**
+     * The LatLng coordinates representing the closest advertised location.
+     * @private {?google.maps.LatLng}
+     */
+    this.activeLocation_ = null;
+    /**
+     * The LatLng coordinates representing the user's current location.
+     * @private {?google.maps.LatLng}
+     */
+    this.currentLocation_ = null;
+    /**
+     * The DirectionsRenderer object that displays directions from
+     * the given request.
+     * @private @const {!google.maps.DirectionsRenderer}
+     */
+    this.directionsRenderer_ = new google.maps.DirectionsRenderer();
+
+    /**
+     * A list of the travel modes supported by the Google Maps API.
+     * @private @const {!array}
+     */
+    this.transportMethods_ = [
```

Comment:
The actual strings can be made a top-level const, then you can iterate over them here (or below) to create them buttons

Reply:
Done